### PR TITLE
感谢<<github 入门与实践>> 提供了这个机会

### PR DESCRIPTION
**Pull request by jiexishede**
```
 Thanks!
  ** Congratulations!**   
```

I can do it.  
Thank you very much.  
I modify the file named "index.html".

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -102,6 +102,13 @@ <h2 id="project_tagline">GitHub实践入门</h2>
 <p class="impression">这本书很不错，对 Git 和 GitHub 的介绍都很详尽，笔调也很轻松，适合闲暇时阅读。（<a href="https://github.com/hbkdsm">@hbkdsm</a>）</p>
 <p class="impression">感谢译者搭建测试平台.2015-08-31.@zh</p>
 <p class="impression">简单易懂，入门必读！</p>
+<p> 对<<github 入门和实践>> 的Pull Request 进行实践。哈哈，by JieXiSheDe。</p>
+    
+<p> 这本书挺好。</p>
+    
+<p class="impression"> 这本书很有意思。by JieXiSheDe(<a href=https://github.com/jiexishede")(@jiexishede) </p>
```
Comment (liubin) [inline index.html]: 这行有个问题吧，href=后面没有双引号。

**Comment by jiexishede** [comment]
I fix the bug for href label

**Comment by jiexishede** [comment]
我修复了 bug

File: index.html [hunk 2]
```diff
@@ -102,6 +102,13 @@ <h2 id="project_tagline">GitHub实践入门</h2>
 <p class="impression">这本书很不错，对 Git 和 GitHub 的介绍都很详尽，笔调也很轻松，适合闲暇时阅读。（<a href="https://github.com/hbkdsm">@hbkdsm</a>）</p>
 <p class="impression">感谢译者搭建测试平台.2015-08-31.@zh</p>
 <p class="impression">简单易懂，入门必读！</p>
+<p> 对《github 入门和实践》的Pull Request 进行实践。哈哈，by JieXiSheDe。</p>
```
Comment (liubin) [inline index.html]: p元素没有class

File: index.html [hunk 3]
```diff
@@ -102,6 +102,13 @@ <h2 id="project_tagline">GitHub实践入门</h2>
...
+    
+<p> 这本书挺好。</p>
```
Comment (liubin) [inline index.html]: 这里也米有class

**Comment by jiexishede** [comment]
I add the impression in the P label .  Thank you for guide me.

**Pull request merged**